src/pydap/tests/test_cas_esgf.py

```diff
@@ -67,13 +67,15 @@ def test_dimension_esgf_query(self):
                                   16837.5, 16838.5, 16839.5, 16840.5, 16841.5])
         assert(np.isclose(data, expected_data).all())
 
-    @unittest.skip("This test should work but does not. "
-                   "An issue should be raised.")
     def test_variable_esgf_query(self):
         session = esgf.setup_session(os.environ.get('OPENID_ESGF'),
                                      os.environ.get('PASSWORD_ESGF'),
                                      check_url=self.url)
-        dataset = open_url(self.url, session=session)
+        # This server does not access retrieval of grid coordinates.
```

Comment:
This comment is nice and descriptive, but could you write up a bit of this information in the user docs? It would be helpful for users to know about this behaviour w/o having to read the code.